presto-main/src/test/java/com/facebook/presto/execution/TestQueryStateMachine.java

```diff
@@ -58,6 +58,22 @@
 
 public class TestQueryStateMachine
 {
+    private static class MockTicker extends Ticker
+    {
+        @ Override
```

Comment:
I'm actually surprise that having a space after `@` works. Let's remove the space.